Core/Core/Inbox/MessageDetails/ViewModel/MessageDetailsViewModel.swift

```diff
@@ -45,6 +46,7 @@ class MessageDetailsViewModel: ObservableObject {
     public let updateState = PassthroughSubject<ConversationWorkflowState, Never>()
 
     // MARK: - Private
+    private var didSentMailSuccessfully = PassthroughSubject<Void, Never>()
```

Comment:
typo
```suggestion
    private var didSendMailSuccessfully = PassthroughSubject<Void, Never>()
```
Please fix in other places, if needed.